openscad.qrc

```diff
@@ -106,6 +106,7 @@
     <file>images/SVG-white.png</file>
     <file>images/PDF.png</file>
     <file>images/PDF-white.png</file>
+    <file>images/Anaglyph-32.png</file>
```

Comment:
We will need 2 icons, one for light, one for dark color desktop scheme. I'll have a look, we did generate most of the others based on an OpenSCAD script. I have to dig that out, I think it might be in some branch.

Reply:
OK. I'll add a "white" icon. Does this also mean there have to be two color schemes for 3D, one with black and one with white background?